index.rst

```diff
@@ -50,7 +50,696 @@
++===========================+========+=================================================+
+| calib_range_id            | uint64 | PRIMARY KEY                                     |
++---------------------------+--------+-------------------------------------------------+
+| first_visit               | uint64 | NOT NULL                                        |
++---------------------------+--------+-------------------------------------------------+
+| last_visit                | uint64 |                                                 |
++---------------------------+--------+-------------------------------------------------+
+| camera_id                 | uint64 | NOT NULL REFERENCES Camera (camera_id)          |
++---------------------------+--------+-------------------------------------------------+
+| physical_filter_id        | uint64 | REFERENCES PhysicalFilter (physical_filter_id)  |
++---------------------------+--------+-------------------------------------------------+
+| UNIQUE (first_visit, last_visit, camera_id, physical_filter_id)                      |
++---------------------------+--------+-------------------------------------------------+
+
++------------------------+--------+-----------------------------------------------------------+
+| *SensorCalibRange*                                                                          |
++========================+========+===========================================================+
+| sensor_calib_range_id  | uint64 | PRIMARY KEY                                               |
++------------------------+--------+-----------------------------------------------------------+
+| first_visit            | uint64 | NOT NULL                                                  |
++------------------------+--------+-----------------------------------------------------------+
+| last_visit             | uint64 |                                                           |
++------------------------+--------+-----------------------------------------------------------+
+| physical_sensor_id     | uint64 | NOT NULL REFERENCES PhysicalSensor (physical_sensor_id)   |
++------------------------+--------+-----------------------------------------------------------+
+| physical_filter_id     | uint64 | REFERENCES PhysicalFilter (physical_filter_id)            |
++------------------------+--------+-----------------------------------------------------------+
+| UNIQUE (first_visit, last_visit, camera_id, physical_sensor_id, physical_filter_id)         |
++------------------------+--------+-----------------------------------------------------------+
+
+Master calibration products are defined over a range of Visits from a given
+Camera, though a range of observation dates could be utilized instead.
+Calibration products may additionally be specialized for a particular
+PhysicalFilter, or may be appropriate for all PhysicalFilters by setting the
+``physical_filter_id`` field to ``NULL``.  Calibration products that are
```

Comment:
Is it possible that a calibration product applies to multiple, but not all, filters?

Reply:
It's much less likely.  I'm okay with forcing that to be modeled by having multiple CalibRanges.